idseq_dag/steps/run_lzw.py

```diff
@@ -84,14 +94,17 @@ def lzw_score(sequence, threshold_readlength):
 
         if seq_length > threshold_readlength:
             # Make sure longer reads don't get excessively penalized
-            adjustment_heuristic = (1 + (seq_length - threshold_readlength) / 1000) # TODO: revisit
-            score = lzw_fraction * adjustment_heuristic
+            #adjustment_heuristic = (1 + (seq_length - threshold_readlength) / 1000) # TODO: revisit
```

Comment:
Any particular reason to keep these commented?

Reply:
No, good point - I will uncomment and remove the previous function.